packages/nimble-components/src/rich-text-mention/users/view/styles.ts

```diff
@@ -15,7 +15,7 @@ export const styles = css`
         box-sizing: border-box;
         font: ${bodyFont};
         color: ${bodyFontColor};
-        white-space: normal;
+        white-space: pre-wrap;
```

Comment:
We had an issue when setting `white-space: normal` as there are `&nbsp;` added for every whitespace being entered by the user which creates a couple of issues: 
1. When a user types `@username` and adds more than three whitespaces and adds `@` again to the editor it will not trigger the popup as the last character is not a whitespace.
2. To filter with a space, the popup closes as the unicode value is different from the normalized string output. Now when the space is added when showing the suggestion popup, the space will also be considered for filtering the list.

![image](https://github.com/ni/nimble/assets/123377523/b31a45e5-0755-4da9-a7b3-defed9a90bed)

By setting the `white-space: pre-wrap;`, it considers only the whitespace and when next time `@` is added it converts to a proper decoration node and shows the popup and filtering as expected with space.